app/collins/models/Asset.scala

```diff
@@ -266,7 +266,8 @@ object Asset extends Schema with AnormAdapter[Asset] with AssetKeys {
     val findLocations = find(PageParams(0, 50, "ASC", "tag"), AttributeResolver.emptyResultTuple, instanceFinder)
       .items
       .collect { case a: Asset => a }
-      .filter { _.tag != MultiCollinsConfig.thisInstance }
+      // filter out the local instance so that we don't perform both a local and a remote query to it
+      .filter(asset => { !MultiCollinsConfig.thisInstance.contains(asset.tag) })
```

Comment:
So, this is a really perverse case, but I think this would mess up if you had a datacenter instance name that was a substring of another datacenter name.  For example, if your datacenters were `DC` and `DC1`.  I think you may want to do an equality match here, maybe even make it case insensitive.